pkg/client/client_suite_test.go

```diff
@@ -8,12 +8,14 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/sirupsen/logrus"
+	"go.uber.org/zap"
 )
 
-var testLogger = logrus.New()
+var testLogger *zap.Logger
 
 func TestTLSBootstrapClient(t *testing.T) {
+	testLogger, _ = zap.NewProduction()
```

Comment:
wonder if we should do this for tests instead?

`zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true))`

Reply:
ahh nvm, this is actually something I think only works with the controller manager zap wrapper "sigs.k8s.io/controller-runtime/pkg/log/zap"`